packages/ubuntu_desktop_installer/lib/pages/choose_your_look_page.dart

```diff
@@ -12,46 +13,66 @@ class ChooseYourLookPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final lang = AppLocalizations.of(context);
+    final width = MediaQuery.of(context).size.width;
+    final settings = Settings.of(context, listen: false);
     return WizardPage(
       header: Text(lang.chooseYourLookPageHeader),
       actions: <WizardAction>[
         WizardAction.back(context),
         WizardAction.next(context),
       ],
       title: Text(lang.chooseYourLookPageTitle),
-      contentPadding: const EdgeInsets.fromLTRB(20, 50, 20, 150),
       content: Center(
-        child: ListView(
-            shrinkWrap: true,
-            scrollDirection: Axis.horizontal,
-            children: <Widget>[
-              Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: OptionCard(
-                  image: FlavorImage.asset('assets/Theme_thumbnails-Light.png'),
-                  title: Text(lang.chooseYourLookPageLightSetting),
-                  body: Text(lang.chooseYourLookPageLightBodyText),
-                  selected: Theme.of(context).brightness == Brightness.light,
-                  onSelected: () {
-                    final settings = Settings.of(context, listen: false);
-                    settings.applyTheme(Brightness.light);
-                  },
+        child: SingleChildScrollView(
```

Comment:
the scroll view can be dropped because children's sizes are calculated below to make them fit

Reply:
I thought so too 😃 but unless the installer won't have a min size defined in my_application.cc you can produce overflow if you make the window very small. That's why I've added the scroll view. Should I remove it and make a second PR for the min sizes?

Reply:
The window has a fixed size in production. :smile:

Reply:
The window is only resizable in debug mode for testing purposes. Layouts should be reasonably scalable even if the window has a certain fixed size right now. So just like you made it calculate the size. :+1: 

If the preferred size changes in the future, the UI is less likely to require much refactoring. It's easiest to prepare those things while working on the page now, instead of coming back to it one year later. :smile: